addons/binding/org.openhab.binding.milight/README.md

```diff
@@ -3,15 +3,46 @@ The openHAB2 Milight binding allows to send commands to multiple Milight bridges
 
 [![openHAB Milight](http://img.youtube.com/vi/zNe9AkQbfmc/0.jpg)](http://www.youtube.com/watch?v=zNe9AkQbfmc)
 
-## Configuration
+## Supported Things
+The Milight Binding supports White, and RGB(W) bulbs.
 
-Just use the auto discovery feature or add a thing for the binding manually
-by providing host and port. Initially a thing for each detected bridge will be created.
+## Discovery
+Version 3+ bridges can be discovered by triggering a search in openHAB's inbox. Found bridges
+will show up an can easily be added as things.
 After a bridge has been added, all possible bridge supported devices will appear
 as new things. Unfortunatelly milight leds are only able to receive so there is
 no real auto detection for single milight leds but only for the briges possible.
 Add the leds you actually configured and hide the rest of the detected things.
 
+## Binding Configuration
+When manually adding an older bridge Type (3-), you have to add configuration information for
+the bridge IP-Address and the listening port.
+
+## Thing Configuration
+Besides adding bridges through Paper-UI, you can also add them manually in your Thing
+configuration file.
+
+    Bridge milight:bridge:ACCF23A6C0B4 [ ADDR="192.168.0.70", PORT=8899 ]
```

Comment:
you should put all those parts in code sections for proper formatting.
Please remember to add empty lines before and after any code setions, so that Jekyll doesn't break the layout.